piv/pcsc_darwin.go

```diff
@@ -36,3 +44,21 @@ func isRCNoReaders(rc C.int) bool {
 	// are available.
 	return false
 }
+
+func (c *scContext) Connect(reader string) (*scHandle, error) {
```

Comment:
Can this be in pcsc_unix.go? I'd like to avoid including the C headers in new files